charts/jenkins/jobs/integration-tests/nuxeo-server-web-ui/server-build-web-ui-release.groovy

```diff
@@ -0,0 +1,34 @@
+pipelineJob('integration-tests/nuxeo-server-web-ui/server-build-web-ui-release') {
+  // Description
+  description('Automatically validate the next Nuxeo version with the latest promoted Web UI version')
+  definition {
+    // Definition
+    cpsScm {
+      scm {
+        // SCM
+        git {
+          // Repositories
+          remote {
+            // Repository URL
+            url('https://github.com/nuxeo/nuxeo-integration-tests')
+            // Credentials
+            credentials('github-branch-source')
+          }
+          // Branches to build
+          branches('*/2021')
+          browser {
+            // Repository browser
+            githubWeb {
+              // URL
+              repoUrl('https://github.com/nuxeo/nuxeo-integration-tests')
+            }
+          }
+        }
+      }
+      // Lightweight checkout
+      lightweight(true)
+      // Script Path
+      scriptPath('nuxeo-server-web-ui/Jenkinsfiles/server-build-web-ui-release.groovy')
```

Comment:
Actually I think we'd like nuxeo-web-ui/Jenkinsfiles/... so that we match the name of the package we're testing in the nuxeo-integration-tests repository, that will probably have some more packages later on, e.g.:
```
nuxeo-integration-tests
      |-- nuxeo-web-ui
      |-- nuxeo-coldstorage
      |-- nuxeo-compound-doculents
      |-- ...
```